Qadir__Work/HTML/Assignment6/style.css

```diff
@@ -0,0 +1,75 @@
+h1 {
+    font-size: x-large;
+}
+
+h1,
+h2 {
+    background-color: rgb(140, 147, 240);
+    border-bottom-left-radius: 10px;
```

Comment:
aap yaha shortcut bhi use kar sakte the

Reply:
No, Because i want only to applying same property if tag sharing same thing.

Reply:
ok